tests/testthat/test-oc_check_status.R

```diff
@@ -4,53 +4,59 @@
 # https://github.com/OpenCageData/opencagedata-misc-docs/blob/master/library-guidelines.md # nolint
 key_402 <- "4372eff77b8343cebfc843eb4da4ddc4" # always returns a 402 responce
 key_403 <- "2e10e5e828262eb243ec0b54681d699a" # always returns a 403 responce
+key_429 <- "d6d0f0065f4348a4bdfe4587ba02714b" # always returns a 429 responce
 
-test_that("oc_check_status returns 402 error if quota exceeded", {
+test_that("oc_check_status returns 400 error if request is invalid", {
   skip_on_cran()
   skip_if_offline()
 
+  # Both shouldn't happen since we oc_check_query
   expect_error(
```

Comment:
so do we need this test? (sorry if my question is naive)

Reply:
That depends on how you define "need", I guess 😉 . Strictly speaking, I suppose one of the tests in test-oc_check_status would suffice, because it is either status 200 or not. However, I'd prefer to test it like this, so we can see whether we get a 40x response where we expect it. Only loosely related, but I wasn't expecting getting 400 errors now with empty queries which made the changes in this PR necessary. 
So long story short: If we want to keep things very concise, we do not need this test, but I don't think it hurts. (and no, not naïve at all). 😄

Reply:
:+1: thanks for the explanation! And feel free to merge :-)